deploy/crds/dynatrace.com_oneagents_crd.yaml

```diff
@@ -58,6 +58,7 @@ spec:
               items:
                 type: string
               type: array
+              x-kubernetes-list-type: set
```

Comment:
Remove the changes for this file. I remember they were breaking some old Kubernetes version. OCP 3.11?